src/app/lib/utilities/makeRelativeUrlPath/index.js

```diff
@@ -0,0 +1,15 @@
+const allowList = ['bbc.com', 'bbc.co.uk', 'bbcrussian.com'];
+
+const makeRelativeUrlPath = urlPath => {
+  if (!urlPath) {
+    return null;
+  }
+  const isBBCDomain = allowList.some(domain => urlPath.includes(domain));
+  if (isBBCDomain) {
+    const relativeUrl = urlPath.replace(/^.*\/\/[^/]+/, '');
+    return relativeUrl.length === 0 ? '/' : relativeUrl;
```

Comment:
Could we use the [URL API](https://developer.mozilla.org/en-US/docs/Web/API/URL) instead? Something like:
```suggestion
    const url = new URL(urlPath);
    return url.pathname;
```
I guess the only thing it wouldn't cover is any hashes added to the URL, but you can grab it with `url.hash`. I'm only suggesting this because of readability over regex

Reply:
Good shout 👍🏼 I wasn't sure if we could use it since IE11 doesn't support `URL`, but we have used it in other areas of the application. Looks a lot cleaner.

Reply:
Doesn't look like params are in the pathname; those will need to be added too with `url.searchparams`. Would be good to include tests for params and hash if used.

Reply:
Updated the function and tests to include checks for the `url.search` and `url.hash` values